test/regress/regress0/error.cvc

```diff
@@ -1,8 +1,7 @@
+% ERROR-SCRUBBER: sed -e '/^[[:space:]]*$/d'
```

Comment:
It is kind of weird that the issue only seems to appear with empty lines. It doesn't hurt to have the error scrubber though. Let's keep it the way you have it right now. I have a close-to-finished Python replacement of this script that should make fixing issues like these hopefully simpler.